docs/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md

```diff
@@ -0,0 +1,24 @@
+The `Amplify/Tools` pod provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Xcode. It integrates into the build process of the IDE to provide a seamsless experience.
+
+First add the pod to your `Podfile`.
+
+```ruby
+pod 'Amplify/Tools'
+```
+
+Then **add a Build Phase** to your app target.
+
+1. Click on your project in the file list, choose your target under TARGETS, click the Build Phases tab.
+2. Add a **New Run Script Phase** by clicking the little plus icon in the top left.
```

Comment:
Suggestions:
* change little plus icon to little `+` icon
* change "top left" to  "top left corner"
* In the instructions on the "setup your project tutorial" we have a setup which says: Update the Run Script build phase title to “Run Amplify Tools”.. maybe we should add that here?